robot/rospackages/src/ping_acknowledgment/scripts/ping_response_server.py

```diff
@@ -5,11 +5,14 @@
 import rospy
 
 def handle_ping(req):
+    with open(os.environ['HOME'] + '/Programming/robotics-prototype/robot/rospackages/src/ping_acknowledgment/scripts/meme.txt', 'r') as meme:
```

Comment:
please don't use this small brain solution. Use a relative path : https://stackoverflow.com/questions/7165749/open-file-in-a-relative-location-in-python

Reply:
Fixed